modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -0,0 +1,91 @@
+
+Trace data contains rich information, and the data is normalized across instrumented languages and frameworks.
+Therefore, additional metrics can be extracted from traces. These metrics are request count, duration, and error count (RED).
+The metrics can be visualized in Jaeger console in the *Monitor* tab.
+
+The metrics are derived from spans in the OpenTelemetry Collector that are scraped from the Collector by the Prometheus deployed in the user-workload monitoring stack.
+The Jaeger UI queries these metrics from the Prometheus endpoint and visualizes them.
+
+== OpenTelemetry Collector configuration
+
+The OpenTelemetry Collector requires configuration of the `spanmetrics` connector that derives metrics from traces and exports the metrics in the Prometheus format.
+
+.OpenTelemetry Collector custom resource for span RED
+[source,yaml]
+----
+kind: OpenTelemetryCollector
+apiVersion: opentelemetry.io/v1alpha1
+metadata:
+  name: otel
+spec:
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true <1>
+  config: |
+    connectors:
+      spanmetrics: <2>
+        metrics_flush_interval: 15s
+
+    receivers:
+      otlp: <3>
+        protocols:
+          grpc:
+          http:
+
+    exporters:
+      prometheus: <4>
+        endpoint: 0.0.0.0:8889
+        resource_to_telemetry_conversion:
+          enabled: true # by default resource attributes are dropped
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          exporters: [otlp, spanmetrics] <5>
+        metrics:
+          receivers: [spanmetrics] <6>
+          exporters: [prometheus]
+----
+<1> Creates the `ServiceMonitor` custom resource to enable scraping of the Prometheus exporter.
+<2> The Spanmetrics connector receives traces and exports metrics.
+<3> The OTLP receiver to receive spans in the OpenTelemetry protocol.
+<4> The Prometheus exporter is used to export metrics in the Prometheus format.
+<5> The Spanmetrics connector is configured as exporter in traces pipeline.
+<6> The Spanmetrics connector is configured as receiver in metrics pipeline.
+
+== Tempo configuration
+
+The `TempoStack` CR needs to enable the monitor tab and set the Prometheus endpoint to Thanos querier service to query the data from the user-defined monitoring stack.
+
+.TempoStack CR with enabled monitor tab
+[source,yaml]
+----
+kind:  TempoStack
+apiVersion: tempo.grafana.com/v1alpha1
+metadata:
+  name: simplest
+spec:
+  template:
+    gateway:
+      enabled: false <1>
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        monitorTab:
+          enabled: true <2>
+          prometheusEndpoint: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091 <3>
+        ingress:
+          type: route
+----
+<1> Gateway is enabled because the monitor tab is not supported thorough the gateway deployment.
+<2> Enables monitoring tab in Jaeger console.
+<3> The Thanos querier service name from user-workload monitoring.
```

Comment:
```suggestion
<1> Gateway must be enabled because the Monitor tab is not supported thorough the gateway deployment.
<2> Enables the monitoring tab in the Jaeger console.
<3> The service name for Thanos Querier from user-workload monitoring.
```